README.md

```diff
@@ -93,7 +93,7 @@ async def subscriber(topic):
         # Handle the event
 
 def main():
-    asyncio.get_event_loop().run_until_complete(subscriber(topic))
+    asyncio.run(subscriber(topic))
```

Comment:
We will have to update the data playground and other examples, but it should be a one line change.